src/testdir/test_popup.vim

```diff
@@ -2067,67 +2085,59 @@ func Test_pum_maxwidth_multibyte()
   call VerifyScreenDump(buf, 'Test_pum_maxwidth_08', {'rows': 8})
   call term_sendkeys(buf, "\<ESC>")
 
-  call StopVimInTerminal(buf)
-endfunc
-
-func Test_pum_maxwidth_with_many_items()
-  CheckScreendump
-
-  let lines =<< trim END
-    func Omni_test(findstart, base)
-    if a:findstart
-      return col(".")
-    endif
-    return [
-      \ #{word: "foo", menu: "fooMenu", kind: "fooKind"},
-      \ #{word: "bar", menu: "barMenu", kind: "barKind"},
-      \ #{word: "baz", menu: "bazMenu", kind: "bazKind"},
-      \ ]
-    endfunc
-    set omnifunc=Omni_test
-  END
-  call writefile(lines, 'Xtest', 'D')
-  let  buf = RunVimInTerminal('-S Xtest', {})
-  call TermWait(buf)
-
-  call term_sendkeys(buf, ":set pummaxwidth=20\<CR>")
-  call term_sendkeys(buf, "S\<C-X>\<C-O>")
-  call VerifyScreenDump(buf, 'Test_pum_maxwidth_with_many_items_01', {'rows': 8})
+  call term_sendkeys(buf, ":set pummaxwidth=14\<CR>")
+  call term_sendkeys(buf, ":let g:change=1\<CR>S\<C-X>\<C-O>")
+  call VerifyScreenDump(buf, 'Test_pum_maxwidth_10', {'rows': 8})
   call term_sendkeys(buf, "\<ESC>")
 
-  call term_sendkeys(buf, ":set pummaxwidth=19\<CR>")
+  " Unicode Character U+2026 but one cell
+  call term_sendkeys(buf, ":set fcs+=ellipsis:…\<CR>")
```

Comment:
This test only tests for using a single character for `ellipsis`. Using multiple characters should also be tested. In particular, there should be a test where `ellipsis` is an asymmetric sequence of characters.

Then a question arises here. Should `ellipsis` be drawn in the reverse order when `'rightleft'` is set? I'm not sure about this.

Reply:
> Should `ellipsis` be drawn in the reverse order when `'rightleft'` is set? I'm not sure about this.

Oh, actually I think it should.

Reply:
right.. ellipsis should also be reversed

Reply:
added